__tests__/parseCliArgs.test.mjs

```diff
@@ -148,14 +127,34 @@ test('It should return true for skipFileGeneration.styled if passing "--skipDesc
 test('It should return true for skipFileGeneration.forceUpdate if passing "--forceUpdate"', () => {
   expect(parseCliArgs(['--forceUpdate'])).toEqual(
     expect.objectContaining({
-      skipFileGeneration: {
-        css: false,
-        description: false,
-        forceUpdate: true,
-        react: false,
-        storybook: false,
-        styled: false
-      }
+      skipFileGeneration: expect.objectContaining({
+        forceUpdate: true
```

Comment:
NOTE: this is the default value in meta/configs.mjs.
I do not think there is a way to set it to false from the CLI arguments?
This flag will only override it if it was set to false in the `.figmagicrc`